shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -49,27 +58,53 @@ protected int getNumOfArgs() {
   }
 
   @Override
-  public int run(CommandLine cl) throws AlluxioException, IOException {
+  public int run(CommandLine cl) throws IOException {
+    // Checks if Alluxio master and client services are running
+    try (CloseableResource<FileSystemMasterClient> client =
+             FileSystemContext.INSTANCE.acquireMasterClientResource()) {
+      try {
+        InetSocketAddress address = client.get().getAddress();
+        List<InetSocketAddress> addresses = Arrays.asList(address);
+        MasterInquireClient inquireClient = new PollingMasterInquireClient(addresses, () ->
+            new ExponentialBackoffRetry(50, 100, 2)
+        );
+        try {
```

Comment:
we can move this try-catch block to be after the previous try-catch block instead of nesting

Reply:
Changed, thanks